Makefile

```diff
@@ -7,6 +7,23 @@ export SPHINX_MOCK_REQUIREMENTS=1
 # install only Lightning Trainer packages
 export PACKAGE_NAME=pytorch
 
+setup:
+	uv pip install -r requirements.txt
+	uv pip install -r requirements/pytorch/base.txt
+	uv pip install -r requirements/pytorch/test.txt
+	uv pip install -r requirements/pytorch/extra.txt
+	uv pip install -r requirements/pytorch/strategies.txt
+	uv pip install -r requirements/fabric/base.txt
+	uv pip install -r requirements/fabric/test.txt
+	uv pip install -r requirements/fabric/strategies.txt
+	uv pip install -r requirements/typing.txt
```

Comment:
can we chain it? with regular pip chain, ensure that all dependencies are satisfied